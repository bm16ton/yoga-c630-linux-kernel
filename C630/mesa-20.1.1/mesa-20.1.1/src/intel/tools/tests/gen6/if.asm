(+f0.0) if(8)   JIP: 84                                         { align16 1Q };
(+f0.0) if(8)   JIP: 32                                         { align1 1Q };
(+f0.0) if(16)  JIP: 32                                         { align1 1H };
(+f0.0.x) if(8) JIP: 18                                         { align16 1Q };
(-f0.0) if(8)   JIP: 12                                         { align1 1Q };
(-f0.0) if(16)  JIP: 12                                         { align1 1H };
