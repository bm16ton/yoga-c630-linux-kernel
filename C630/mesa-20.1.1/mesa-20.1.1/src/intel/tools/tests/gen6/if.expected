22 01 61 00 8f 10 54 00 04 00 0e 00 04 00 0e 00
22 00 61 00 8f 10 20 00 00 00 00 00 00 00 00 00
22 00 81 00 8f 10 20 00 00 00 00 00 00 00 00 00
22 01 62 00 8f 10 12 00 04 00 0e 00 04 00 0e 00
22 00 71 00 8f 10 0c 00 00 00 00 00 00 00 00 00
22 00 91 00 8f 10 0c 00 00 00 00 00 00 00 00 00
